### PR TITLE
add node affinity to the UI

### PR DESCRIPTION
**Pull request by yashbhutwala**
add node-affinity in ui

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/665#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by yashbhutwala** [comment]
Closing this for now, first merge #666 because of dependency then will re-open a separate ui-only PR

**Comment by operator-r2d2** [comment]
@yashbhutwala: PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by rrati** [comment]
@msenmurugan PTAL.  Looks ok to me other than needed a rebase

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/665#issuecomment-512511843" title="Approved">rrati</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/665#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [rrati,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: cmd/operator-ui/templates/blackducks/_form.html [hunk 1]
```diff
@@ -71,6 +71,11 @@
       <li value="<%= environ %>"><%= environ %></li>
       <% } %>
     </ul>
+    <input type="hidden" id="blackduck-View.NodeAffinities" name="View.NodeAffinities" value="" />
+    <% } %>
```
Comment (msenmurugan) [inline cmd/operator-ui/templates/blackducks/_form.html]: @yashbhutwala why we need this close button?

Comment (yashbhutwala) [reply cmd/operator-ui/templates/blackducks/_form.html]: good catch, fixed.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/665#pullrequestreview-263251396" title="LGTM">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/665#issuecomment-512511843" title="Approved">rrati</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/665#" title="Author self-approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/master/OWNERS)~~ [msenmurugan,rrati,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@rrati: once the present PR merges, I will cherry-pick it on top of release-2019.6.x in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/665#issuecomment-512523853):

>/cherrypick release-2019.6.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@rrati: #665 failed to apply on top of branch "release-2019.6.x":
```Applying: add node affinity to the UI
error: Failed to merge in the changes.
Using index info to reconstruct a base tree...
M	cmd/operator-ui/go.mod
M	cmd/operator-ui/go.sum
Falling back to patching base and 3-way merge...
Auto-merging cmd/operator-ui/go.sum
CONFLICT (content): Merge conflict in cmd/operator-ui/go.sum
Auto-merging cmd/operator-ui/go.mod
CONFLICT (content): Merge conflict in cmd/operator-ui/go.mod
Patch failed at 0001 add node affinity to the UI

```

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/665#issuecomment-512523853):

>/cherrypick release-2019.6.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**